pkg/users/client.go

```diff
@@ -238,6 +249,65 @@ type ListAuthFactorsResponse struct {
 	ListMetadata common.ListMetadata `json:"listMetadata"`
 }
 
+type Invite struct {
+	ID             string     `json:"id"`
+	Email          string     `json:"email"`
+	State          TokenState `json:"state"`
+	AcceptedAt     string     `json:"accepted_at,omitempty"`
+	RevokedAt      string     `json:"revoked_at,omitempty"`
+	Token          string     `json:"token"`
+	OrganizationID string     `json:"organization_id,omitempty"`
+	ExpiresAt      string     `json:"expires_at"`
+	CreatedAt      string     `json:"created_at"`
+	UpdatedAt      string     `json:"updated_at"`
+}
+
+type CreateInviteOpts struct {
+	Email          string `json:"email"`
+	OrganizationID string `json:"organization_id,omitempty"`
+	ExpiresInDays  int    `json:"expires_in_days,omitempty"`
+	InviterUserID  string `json:"inviter_user_id,omitempty"`
+}
+
+type RevokeInviteOpts struct {
+	InviteID string
+}
+
+// ListInvites contains the response from the ListInvites call.
+type ListInvitesResponse struct {
+	// List of Invites
+	Data []Invite `json:"data"`
+
+	// Cursor to paginate through the list of invites
+	ListMetadata common.ListMetadata `json:"listMetadata"`
+}
+
+type ListInvitesOpts struct {
+	OrganizationID string `json:"organization_id,omitempty"`
+
+	Email string `json:"email,omitempty"`
+
+	// Maximum number of records to return.
+	Limit int `url:"limit"`
+
+	// The order in which to paginate records.
+	Order Order `url:"order,omitempty"`
+
+	// Pagination cursor to receive records before a provided User ID.
+	Before string `url:"before,omitempty"`
+
+	// Pagination cursor to receive records after a provided User ID.
+	After string `url:"after,omitempty"`
+}
+
+type GetInviteOpts struct {
+	InviteID string
+}
```

Comment:
```suggestion
type GetInviteOpts struct {
        // Invite unique identifier
	Invite string `json:"id"`
}
```
looks like in other `Get*Opts` cases we use the object name here instead, so let's follow that convention:
https://github.com/workos/workos-go/blob/c831fd777f3fa50f092ab4cb880bb768f33a3c14/pkg/users/client.go#L86-L90

(sorry i didn't catch this before)

Reply:
idk if `json:"id"` is correct btw i just copy/pasta-ed from the user case

Reply:
and i suggest using that same convention for `RevokeInviteOpts`

Reply:
yeah this is a contested naming convention at times, sometimes we call it `User` other times we call it `UserID` or just `ID` (this particular package is going with the `User` style convention but we've debated it in slack). I'll go ahead and call it `User` style for package consistency

Reply:
Also it doesn't need the JSON string here because it's being passed in in the URL.

Reply:
JSON tags are irrelevant in the opts - just the response where there's JSON encoding/decoding.